### PR TITLE
infra: build_status: improve fuzz introspector overview page

### PR DESCRIPTION
**Pull request by DavidKorczynski**
Extend the page displaying Fuzz Introspector projects with more information. Primarily, reachability and coverage information and then wrapped it in a table where searching and sorting is possible.

Example page: https://davidkorczynski.github.io/staticpages/fuzz_index.html

Signed-off-by: David Korczynski <david@adalogics.com>

### CODE REVIEW

File: infra/build/build_status/update_build_status.py [hunk 1]
```diff
@@ -29,6 +29,7 @@
 import build_and_run_coverage
 import build_lib
 import build_project
+import fuzz_introspector_page_gen
```
Comment (jonathanmetzman) [inline infra/build/build_status/update_build_status.py:32]: put this after 35 please (those imports aren't really compliant with our style as is, but this is more alphabetical).

**Comment by DavidKorczynski** [comment]
The infra tests seems unrelated:
```
Step 5/8 : RUN git clone --depth 1 https://gitlab.gnome.org/GNOME/libxml2.git
 ---> Running in 3697137aab34
Cloning into 'libxml2'...
fatal: unable to access 'https://gitlab.gnome.org/GNOME/libxml2.git/': The requested URL returned error: 503

----------------------------- Captured stderr call -----------------------------
```

File: infra/build/build_status/fuzz_introspector_page_gen.py [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+from bs4 import BeautifulSoup
+
+TABLE_HEAD = """<table id="mainTable">
+<thead>
+            <tr>
+      <th title="Fuzzer key. Usually fuzzer executable file">
+       Project Report
+      </th>
+      <th title="Number of fuzzers for the project.">
+       Fuzzer count
```
Comment (oliverchang) [inline infra/build/build_status/fuzz_introspector_page_gen.py:28]: Nit: can we rename this to "fuzz target" to be consistent with our other documentation?

**Review by oliverchang — APPROVED** [review]
thanks! just one very minor comment.

File: infra/build/build_status/fuzz_introspector_page_gen.py (hunk 2 above)
Comment (oliverchang) [reply infra/build/build_status/fuzz_introspector_page_gen.py]: I edited this myself.

**Comment by DavidKorczynski** [comment]
fyi: to test the html generation locally the approach I took was use the script I removed in the commit here: https://github.com/google/oss-fuzz/pull/9538/commits/2ae3a2b738b860b5d31f142f7d6cd1de62908370

Place that script in the same directory as the added `.py` file and run it, which will print out the html page.

**Comment by oliverchang** [comment]
infra tests indeed look unrelated. https://gitlab.gnome.org/GNOME/libxml2 is also down for me.

**Pull request merged**